flink-runtime/src/test/java/org/apache/flink/runtime/registration/RegisteredRpcConnectionTest.java

```diff
@@ -23,45 +23,39 @@
 import org.apache.flink.runtime.rpc.RpcService;
 import org.apache.flink.runtime.rpc.TestingRpcService;
 import org.apache.flink.types.Either;
-import org.apache.flink.util.TestLogger;
 
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.slf4j.LoggerFactory;
 
 import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Executor;
 
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** Tests for RegisteredRpcConnection, validating the successful, failure and close behavior. */
-public class RegisteredRpcConnectionTest extends TestLogger {
+class RegisteredRpcConnectionTest {
```

Comment:
You may need to add `@ExtendWith(TestLoggerExtension.class)` for `RegisteredRpcConnectionTest` after removing `extends TestLogger`